tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -25,6 +25,7 @@ profiles:
     oidc_config: "managed"
     auditlog_forward: true
     admin_enabled: false
+    volume_size: 150
```

Comment:
Let us use the edge value, which is the smallest, to do testing.

Reply:
Updated to `75`.